src/views/v2/list/top-bar/datepicker.php

```diff
@@ -34,7 +33,7 @@
 }
 
 $selected_start_datetime = strtotime( $selected_start_date_value );
-$is_now = date( Dates::DBDATEFORMAT, $selected_start_datetime ) === date( Dates::DBDATEFORMAT, strtotime( $default_start_date ) );
+$is_now =  Dates::build_date_object( $selected_start_datetime )->format( Dates::DBDATEFORMAT ) === Dates::build_date_object( $default_start_date )->format( Dates::DBDATEFORMAT );
```

Comment:
* :warning: Equals sign not aligned with surrounding assignments; expected 18 spaces but found 1 space
* :no_entry_sign: Expected 1 space after "="; 2 found
* :no_entry_sign: Expected 1 space after "="; 2 found

<sub>hash: e2ef8b32c6de2ad6af32fbbc45627367</sub>